### ISSUE TITLE
selectedRangeStartStyle and selectedRangeEndStyle  styling

### ISSUE THREAD

**Issue by Jere2357** [stephy/CalendarPicker#234]
# Issue on styling selectedRangeStartStyle and selectedRangeEndStyle

-  I'm using Expo and I can't style my _selectedRangeStartStyle_ and _selectedRangeEndStyle_  into box figured, it's always rounded.

- I hope someone has solution.

- I want to style my _selectedRangeStartStyle_ and _selectedRangeEndStyle_ into this:

![image](https://user-images.githubusercontent.com/54886032/93558636-74f51300-f9b0-11ea-94dd-b17cbc1fe0bf.png)

**Comment by dave-martinez** [comment]
I have encountered this problem.

My plan was to make the range start and end to circle but apparently, there is a wrapper around the start and end date range that handles the border radius.

Hoping that we expose the wrapper prop and allow custom styling for range start and end dates.

**Comment by Jere2357** [comment]
Yeah, @dave-martinez  I also observed that the text that the _selectedRangeStartStyle_ and _selectedRangeEndStyle_ has no _color_ style prop in order to make the start and end text color different from other selected date range.

**Comment by peacechen** [comment]
Please submit a PR and I'll review right away 👍

**Comment by peacechen** [comment]
Reviewing the code, it should be possible to achieve what you want.
https://github.com/stephy/CalendarPicker/blob/master/CalendarPicker/Day.js#L207

For the text color of the selected day(s), use `selectedDayTextStyle`. A bug with that prop was fixed in 7.0.7.

Props `selectedRangeStartStyle` and `selectedRangeEndStyle` should override the defaults for the container styling.
